ome/omero-web
# Script delete

NB: This is on top of #126.
Adds support for deleting scripts from webclient.

![Screenshot 2020-02-19 at 13 33 21](https://user-images.githubusercontent.com/900055/74838994-6c563480-531c-11ea-83e7-d9f49cc7b135.png)

To test:
 - Login as Admin
 - Choose Delete option
 - Try deleting 1 or more scripts
 - On completion, they should be removed from the webclient scripts menu.
